site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/blr-related-works/components/Filter.js

```diff
@@ -0,0 +1,87 @@
+// This file is part of Zenodo.
+// Copyright (C) 2023 CERN.
+//
+// Zenodo is free software; you can redistribute it
+// and/or modify it under the terms of the GNU General Public License as
+// published by the Free Software Foundation; either version 2 of the
+// License, or (at your option) any later version.
+//
+// Zenodo is distributed in the hope that it will be
+// useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Zenodo; if not, write to the
+// Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+// MA 02111-1307, USA.
+//
+// In applying this license, CERN does not
+// waive the privileges and immunities granted to it by virtue of its status
+// as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+import React from "react";
+import { PropTypes } from "prop-types";
+import { Dropdown, Label, Button, Icon } from "semantic-ui-react";
+
+export const FilterContainer = ({ agg, containerCmp, updateQueryFilters }) => {
+  const clearFacets = () => {
+    if (containerCmp.props.selectedFilters.length) {
+      updateQueryFilters([agg.aggName, ""], containerCmp.props.selectedFilters);
+    }
+  };
+
+  return (
+    <div className="flex align-items-center">
+      <div>{containerCmp}</div>
+      <div>
+        <Button onClick={clearFacets} content="Reset filters" />
+      </div>
+    </div>
+  );
+};
+
+FilterContainer.propTypes = {
+  agg: PropTypes.object.isRequired,
+  updateQueryFilters: PropTypes.func.isRequired,
+  containerCmp: PropTypes.node.isRequired,
+};
+
+export const Filter = ({ valuesCmp }) => {
+  return (
+    <Dropdown text="Filter by type" button>
```

Comment:
Comment: By using the `text` prop, the user will not see what type of filter is selected unless he opens the dropodwn. I find it more intuitive if once we have selected the filter it's value is displayed. 
```suggestion
    <Dropdown placeholder="Filter by type" button>
```

Reply:
Thanks! I do agree, although this dropdown is a bit different since it works like the filters on the regular search (with checkboxes), so multiple options can be checked.

Reply:
Right! Good point! In that case I would just display some text mentioning how many filters are selected like "X filters selected" so that the user is aware that there are filters selected when the dropdown is closed. However this can be addressed in another issue, if we do so please create the issue!

Reply:
Thanks! I will test it!